docs/configuration.md

```diff
@@ -429,10 +441,10 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 <tr>
   <td><code>spark.shuffle.io.retryWait</code></td>
-  <td>5</td>
+  <td>5s</td>
   <td>
-    (Netty only) Seconds to wait between retries of fetches. The maximum delay caused by retrying
-    is simply <code>maxRetries * retryWait</code>, by default 15 seconds.
+    (Netty only) How long to wait between retries of fetches. The maximum delay caused by retrying
+    is simply <code>maxRetries * retryWait</code>, by default 5 seconds.
```

Comment:
wrong! I filed a JIRA for this specifically because the wording here is confusing. In fact, this was fixed in #4769 but somehow the changes there got lost? @srowen are you aware of this? I suspect the author of #3863 accidentally dropped your changes when resolving merge conflicts.

In any case, this should remain 15 seconds (this refers to the product, not the value of the config itself).